game/script-jokes.rpy

```diff
@@ -631,3 +659,74 @@ label joke_con_crete:
     extend 1nchgn "-crete!"
 
     return
+
+label joke_footless_snakes:
+    n 1ulraj "Hey,{w=0.2} [player]...{w=1}{nw}"
+    extend 1tnmfl " have you ever wondered how to measure snakes?"
+    n 1uwdaj "Like they are so long!"
+    n 1fsqcs "What measurement could be possibly the best?"
+    n 1fcsss "Well,{w=0.2} my guess would be in inches or centimetre...{w=0.5}"
+    n 1nsrss "Because they...{w=1}{nw}"
+    extend 1nchgn " have no {i}feet{/i}!"
+
+    return
+
+label joke_bigger_ball:
+    n 1ullbo "You know..."
+    n 1nslss "When I was playing sports lately..."
+    n 1uwdaj "I wondered why the ball keeps getting bigger!{w=0.5}"
+    n 1flrfl "Then it{w=1}{nw}" 
+    extend 1fsqcs " {i}hit{/i} me!"
+
+    return
+
+label joke_meeting_walls:
+    n 1fsqfl "...What did the wall say to the other wall?"
+    n 1fcspo "..."
```

Comment:
I love the use of the pause here to show her exasperation! I think I'd suggest using the `embarrassed` mouth type with perhaps the `sigh` emote rather than the pout, as I feel it would work a little better in this context.